src/aat/resources/features/F-1001/S-1002.td.json

```diff
@@ -0,0 +1,47 @@
+{
+  "title": "must successfully receive a document am data with an empty list of permissions",
+
+  "_guid_": "S-1002",
+  "_extends_": "F-1001_Test_Data_Base",
+
+  "specs": [
+    "an active Solicitor profile in CCD with a specific variation of ACLs on a case type",
+    "an active Solicitor profile having full permissions on the same case type",
+    "has the case id just created where a document id is associated",
+    "contains active Solicitor profile who does not have any access",
+    "contains the document am data and an empty list of permissions"
+  ],
+
+  "request": {
+    "pathVariables": {
+      "docId": "76976634-374f-472b-a819-a3a938b4cc83"
+    }
+  },
+
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
+    "body": {
+      "ResponseCode": "200",
+      "status": "OK",
+      "timeStamp": "[[ANYTHING_PRESENT]]",
+      "ResponseBody": {
+        "caseDocumentMetadata": {
+          "caseId": "1234567890123456",
+          "caseTypeId": "123",
+          "jurisdictionId": "BEFTA2",
+          "documents": [
+            {
```

Comment:
What is this ResponseBody inside body, here?

Reply:
Updated.